Wasm/Engine.lean

```diff
@@ -403,10 +465,10 @@ mutual
         if n = 0 then pure () else runOp (.br li)
 end
 
-def runDo (_s : Store m)
+def runDo (s : Store m)
           (f : FunctionInstance m)
           (σ : Stack)
-          : Except EngineErrors Stack := do
+          : Except EngineErrors (Globals × Stack) := do
```

Comment:
Do you think this will scale as we implement more of Wasm?

Reply:
That depends on what we need to return from every invocation. I don't _think_ it will be much more than globals, but of course if it does become unwieldy, we can come up with some other monadic solution.